service/routes.py

```diff
@@ -260,36 +264,60 @@ def post(self):
 
         logger.info("Shopcart with ID [%s] created.", shopcart.id)
 
-        location_url = url_for("get_shopcart", shopcart_id=shopcart.id, _external=True)
+        #location_url = url_for("get_shopcart", shopcart_id=shopcart.id, _external=True)
+        location_url = api.url_for(ShopcartResource, shopcart_id=shopcart.id, _external=True)
 
         return shopcart.serialize(), status.HTTP_201_CREATED, {"Location": location_url}
 
 
 ######################################################################
-# G E T  S H O P C A R T
+# G E T  A N D  D E L E T E  S H O P C A R T
 ######################################################################
-@app.route("/shopcarts/<int:shopcart_id>", methods=["GET"])
-def get_shopcart(shopcart_id):
+@api.route('/shopcarts/<int:shopcart_id>')
+@api.param('shopcart_id', 'The Shopcart identifier')
+class ShopcartResource(Resource):
     """
-    Gets information about a Shopcart
-    This endpoint will get information about a shopcart
+    ShopcartResource class
+    Allows the manipulation of a single shopcart
+    GET /shopcart/{id} - Returns a shopcart with the id
+    DELETE /shopcart/{id} -  Deletes a shopcart with the id
     """
-    logger.info("Request to get information of a shopcart")
-    check_content_type("application/json")
 
-    shopcart = Shopcart.find(shopcart_id)
-    if shopcart is None:
-        logger.info("Shopcart with ID [%s] not found.", shopcart_id)
-        return not_found("Shopcart with ID [%s] not found." % shopcart_id)
+    @api.doc('get_shopcart')
+    @api.response(404, 'Shopcart not found')
```

Comment:
I think we also want to include the 200 response here?

Reply:
Added